docs/01-overview/main-areas/service-mesh/smsh-03-istio-sidecars-in-kyma.md

```diff
@@ -18,7 +18,7 @@ Another security benefit of having a sidecar proxy is that you can perform [requ
 
 ## Observability
 
-Furthermore, Istio proxies improve tracing: Istio performs global tracing and forwards the data to [Kyma's tracing component](../../../01-overview/main-areas/observability/obsv-03-tracing-in-kyma.md) using the [Zipkin protocol](https://zipkin.io). Learn more about the process in [Tracing Architecture](../../../05-technical-reference/00-architecture/obsv-03-architecture-tracing.md).
+Furthermore, Istio proxies improve tracing: Istio performs global tracing and forwards the data to a tracing backend. One can bring own tracing backend or one can deploy [Jaeger](https://github.com/kyma-project/examples/tree/main/jaeger) inside the cluster using the [Zipkin protocol](https://zipkin.io). Learn more about  in [Tracing Architecture](../../main-areas/telemetry/telemetry-03-traces.md).
```

Comment:
Let's replace Zipkin by OTLP here. That's what all the defaults for Istio tracing expect now and also the preferred way with Jaeger.